ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/teaser/clientlibs/less/teaser.less

```diff
@@ -0,0 +1,86 @@
+.teaser{
+    position:relative;
+    .cmp-teaser__image{
+        img {
+            max-width: 100%;
+            height: auto;
+            margin: 0;
+        }
+    }
+ 
+    .cmp-teaser__content{               
+        position: absolute;
+        z-index: 99;
+        top: 10%;
+        font-family: sans-serif, Arial;
+        color: #fff;
+        left: 7%;
+        width:65%;
+        
+        .cmp-teaser__title{
+            text-transform:uppercase;
+            font-size:58px;
+            font-weight:700;
+            margin: 0;
+            line-height: 1.2;
+            padding: 0; 
+ 
+                    @media screen and (min-width: 768px) and (max-width: 1024px) {
+                        font-size:42px;
+                    }
+ 
+                    @media screen and (max-width: 767px) {
+                        font-size:34px;
+                    }
+ 
+                    .cmp-teaser__title-link{
+                        color:#fff;
+                        text-decoration:none;
+ 
+                        &:hover{
+                            color:#fff;
+                            text-decoration:none;
+                        }
+                    }
+        }
+        
+        .cmp-teaser__description{
+            font-size:32px;
+
```

Comment:
@Rajamouli-hcl  @ashishgoyalweb : can see lot of whitespace .. please refactor

Reply:
changes updated